components/wizard/WizardAdminAccount.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <v-form ref="form" v-model="validInputs">
+    <v-text-field
+      v-model="admin.Name"
+      outlined
+      :label="$t('username')"
+      type="username"
+      required
+    ></v-text-field>
+    <v-text-field
+      v-model="admin.Password"
+      outlined
+      :label="$t('password')"
+      :rules="[(v) => !!v || $t('passwordRequired')]"
+      required
+      :append-icon="showPassword ? 'mdi-eye-off' : 'mdi-eye'"
+      :type="showPassword ? 'text' : 'password'"
+      @click:append="() => (showPassword = !showPassword)"
+    ></v-text-field>
+    <v-text-field
+      v-model="passwordCheck"
+      outlined
+      :label="$t('confirmPassword')"
+      required
+      type="password"
+      :rules="[(v) => (!!v && v === admin.Password) || $t('bothPasswordsSame')]"
```

Comment:
cc @MrTimscampi are you always adding the rules inline? I've found moving them to an object in the component to be much more managable over time, especially when you want more than one per item. You can also create a utility somewhere to handle common rules and add them wherever you want to reduce code duplication, which helps for more complex rules.

Reply:
For now, we do a combination of both.

I've been thinking about moving validation to something like [VeeValidate](https://github.com/logaretm/vee-validate), though.